File: kicksy/lib/model/employee.dart
class Employee {
  final int emp_code;
  final String password;
  final String division;
  final String grade;

  Employee({
    required this.emp_code,
    required this.password,
    required this.division,
    required this.grade,
  });

  Employee.fromMap(Map<String, dynamic> res)
    : emp_code = res['emp_code'],
      password = res['password'],
      division = res['division'],
      grade = res['grade'];
}

File: python_FASTAPI/employee.py
from fastapi import APIRouter, Form
import pymysql


router = APIRouter()

def connect():
    return pymysql.connect(
        host="192.168.50.4",
        user = "team",
        password="qwer1234",
        db="kicksy",
        charset="utf8",
    )



@router.get("/{emp_code}")
async def selectAll(emp_code : int):
    conn = connect()
    curs = conn.cursor()

    curs.execute("select * from employee where emp_code = %s",(emp_code,))
    rows = curs.fetchall()
    conn.close()

    result = [{"emp_code":row[0],"password":row[1],"division":row[2],"grade":row[3]} for row in rows]
    return {'results':result}

@router.get("/")
async def selectAll():

    conn = connect()
    curs = conn.cursor()

    curs.execute("select * from employee")
    rows = curs.fetchall()
    conn.close()

    result = [{"emp_code":row[0],"password":row[1],"division":row[2],"grade":row[3]} for row in rows]
    return {'results':result}
